nvflare/private/fed/server/fed_server.py

```diff
@@ -795,6 +794,9 @@ def _turn_to_hot(self):
                                     snapshot=snapshot,
                                     fl_ctx=fl_ctx,
                                 )
+                            restored_job_ids.append(job_id)
+            with self.engine.new_context() as fl_ctx:
+                self.engine.job_runner.update_abnormal_finished_jobs(restored_job_ids, fl_ctx=fl_ctx)
```

Comment:
A question:

1. For the "restored" jobs, do they have same job ID as before?
    - If they do, this logic will mark all those jobs as "FINISHED_ABNORMAL"

Reply:
Ok, I found they do have the same ID as before.

But since we try to "restore jobs first", then we mark things that are not restored to be "FINISHED_ABNORMAL"